logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/Message.kt

```diff
@@ -287,6 +288,24 @@ sealed interface Message {
     sealed class EditStatus {
         object NotEdited : EditStatus()
         data class Edited(val lastTimeStamp: String) : EditStatus()
+
+        override fun toString(): String = when (this) {
```

Comment:
I added this toString function. IDK if this breaks something as before it would just used a default string rep